.azure-pipelines-templates/deploy_aci.yml

```diff
@@ -37,18 +41,47 @@ jobs:
         name: wait_for_image
         displayName: "Wait for CI Container Building"
 
+      - ${{ if eq(parameters.deploy_attestation_container, true) }}:
+          - script: |
+              set -ex
+              cd attestation-container
+              docker build -t attestation-container .
+              az acr login --name attestationcontainerregistry
+              login_server=$(az acr show --name attestationcontainerregistry --query loginServer --output tsv)
+              image_version="ci-$(echo $(Build.BuildNumber) | sed 's/\.//g')"
+              docker tag attestation-container $login_server/attestation-container:$image_version
+              docker push $login_server/attestation-container:$image_version
+              echo "##vso[task.setvariable variable=ver;isoutput=true]$image_version"
+            name: build_and_push_attestation_container
+            displayName: "Build and Push Attestation Container"
+
       - script: |
           set -ex
-          python3.8 scripts/azure_deployment/arm_template.py deploy aci --subscription-id $(CCF_AZURE_SUBSCRIPTION_ID) --resource-group ccf-aci --aci-type dynamic-agent --deployment-name ci-$(Build.BuildNumber) --aci-image ccfmsrc.azurecr.io/ccf/ci:pr-$(wait_for_image.gitSha) > ~/aci_ips
-          echo "##vso[task.setvariable variable=ipAddresses;isOutput=true]`cat ~/aci_ips`"
+          if [ ${{ parameters.deploy_attestation_container }} == "True" ]; then
+            resource_group="attestationContainer"
+          else
+            resource_group="ccf-aci"
+          fi
```

Comment:
Can't we specify this at the calling point parameters? So rather than `deploy_attestation_container: False` in `azure_pipelines_snp`, could set a `resource_group: ccf-aci` parameter there?

Reply:
As discussed [here](https://github.com/microsoft/CCF/pull/4694#discussion_r1065593638), I'll revert the change to share `deploy_aci.yml` with `CCF SNP` and `attestation container` pipeline.